Sources/MountebankSwift/Api/Mountebank.swift

```diff
@@ -170,7 +187,7 @@ public struct Mountebank {
     /// - Throws: `MountebankValidationError` if connection between the client and server fails in some way.
     @discardableResult
```

Comment:
In the comment block it says 
`Returns: The deleted Imposter`

Should be 

`- Returns: The deleted Imposters`